.storybook/gardenTheme.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { create } from '@storybook/theming/create';
+
+const theme = {
+  fontBase: DEFAULT_THEME.fonts.system,
+  fontCode: DEFAULT_THEME.fonts.mono
+};
+
+export const managerTheme = create({
+  brandTitle: 'CKEditor / Zendesk Garden',
+  brandUrl: 'https://zendeskgarden.github.io/react-components/storybook',
```

Comment:
I think this should be https://zendeskgarden.github.io/ckeditor/ right?

Reply:
Nice find. Updated.